src/components/IssueType.jsx

```diff
@@ -19,7 +19,8 @@ const IssueType = ({ name, formik = {}, items, handleAddressChange, handleAddres
 					<p className="smallest">{values.requestType} > {values.subRequestType}</p>
 				</div>
 			</div>
-			<div className="address-search-wrapper">
+			
+			<div className={hasError && isTouched ? "error address-search-wrapper" : "address-search-wrapper"}>
```

Comment:
this can be simplified to avoid duplicating static strings.

```jsx
<div className={`address-search-wrapper${hasError && isTouched ? " error" : ""}`}>
```

we also have a package called [classnames](https://www.npmjs.com/package/classnames) in which you can build a set of classes. Below would be the same as the example above.

```jsx
import classNames from 'classnames';

const addressSearchCssClasses = classNames('address-search-wrapper', { 'error': hasError && isTouched });